design-tokens/tokens/primitive/primitives.default.json

```diff
@@ -2309,7 +2477,7 @@
           }
         }
       },
-      "xxlarge": {
+      "3xlarge": {
         "$type": "number",
         "$value": "{base.dimension.76800}",
```

Comment:
Related to base.dimension tokens above, I was thinking this should point to `base.dimension.38400` and have the base dimensions tweaked.

Reply:
Adjusted.